lib/widgets/entity_context_menu.dart

```diff
@@ -28,6 +30,8 @@ class EntityContextMenu extends StatelessWidget {
         if (onCopy != null)
           const SingleActivator(LogicalKeyboardKey.keyC, control: true):
               onCopy!,
+        if (onTrash != null)
+          const SingleActivator(LogicalKeyboardKey.cancel): onTrash!,
```

Comment:
I think we should rather use the delete key for this action (as it's the standard acoss all file expolorers)

Reply:
oh i thought that was the proper one, i meant to have it be delete, will correct